application/app/services/dataverse/collection_service.rb

```diff
@@ -29,11 +29,12 @@ def find_collection_by_id(id)
       CollectionResponse.new(response.body)
     end
 
-    def search_collection_items(collection_id, page: 1, per_page: 10, include_collections: true, include_datasets: true)
+    def search_collection_items(collection_id, page: 1, per_page: 10, include_collections: true, include_datasets: true, query: nil)
       start = (page-1) * per_page
       type_collection = include_collections ? "&type=dataverse" : ""
       type_dataset = include_datasets ? "&type=dataset" : ""
-      query_string = "q=*&show_facets=true&sort=date&order=desc&show_type_counts=true&per_page=#{per_page}&start=#{start}#{type_collection}#{type_dataset}&subtree=#{collection_id}"
```

Comment:
These URLs are becoming too complicated. Should we use a URL builder from Rails to avoid string concatenation?